packages/react-app/src/components/warnings/GnosisSafeWarning.jsx

```diff
@@ -0,0 +1,24 @@
+import { Alert, AlertIcon, Flex, Text } from '@chakra-ui/react';
+import { useWeb3Context } from 'contexts/Web3Context';
+import React from 'react';
+
+export const GnosisSafeWarning = ({ noShadow = false }) => {
+  const { isGnosisSafe } = useWeb3Context();
+
+  if (!isGnosisSafe) return null;
+  return (
+    <Flex align="center" direction="column" w="100%" mb="4">
+      <Alert
+        status="warning"
+        borderRadius={5}
+        boxShadow={noShadow ? 'none' : '0px 1rem 2rem rgba(204, 218, 238, 0.8)'}
+      >
+        <AlertIcon minWidth="20px" />
+        <Text fontSize="small">
+          It is mandatory to set an alternative recipient address when
```

Comment:
Consider to extend the message to say that usually the user running the OB UI as a Gnosis Safe App needs to specify the address of the Gnosis Safe on another side of the bridge.

Reply:
Sure. Can do!